src/test/scala/com/atomist/rug/kind/python3/RequirementsTxtTypeUsageTest.scala

```diff
@@ -17,12 +14,7 @@ abstract class RequirementsTxtTypeUsageTest extends FlatSpec with Matchers {
     )
 
   def exec(program: String, as: ArtifactSource, params: Map[String, String] = Map()): ModificationAttempt = {
-    val runtime = new DefaultRugPipeline(DefaultTypeRegistry)
-    val as = new SimpleFileBasedArtifactSource("", StringFileArtifact("editor/LineCommenter.rug", program))
-    val eds = runtime.create(as)
-    assert(eds.size === 1)
-    val pe = eds.head.asInstanceOf[ProjectEditor]
-    pe.modify(as, SimpleParameterValues( params))
+    ???
```

Comment:
The entire class is abstract and therefore the tests are not executed. It should probably be deleted but not worth blocking merge